proofs/signatures/er.plf

```diff
@@ -64,13 +64,17 @@
                                            (holds cln)))
                   (holds cln)))))
 
-; Takes a `list` of literals and a clause, `suffix`, adds the suffix to each
-; literal and returns a list of clauses as a `cnf`.
-(program clause_add_suffix_all ((list clause) (suffix clause)) cnf
-         (match list
-                (cln cnfn)
-                ((clc l rest) (cnfc (clc l suffix)
-                                    (clause_add_suffix_all rest suffix)))))
+; Represents multiple conjoined clauses.
+; There is a list, `heads` of literals, each of which should is suffixed by the
+; same `tail`.
+(declare common_tail_cnf_t type)
```

Comment:
What does "_t" stand for? I expected to see "_n" as in other places.

Reply:
It is suppose to stand for "type". My idea here was to define a new type (`common_tail_cnf_t`) and a constructor for that type (`common_tail_cnf`).
Note that this type is **not** intended to be recursive -- it's just a pair of `clause`s -- the first clause is a list of literals and the second is a clause. The idea is that  `common_tail_cnf head tail` represents a cnf in which each clause begins with a literal from `head`, and ends with the same suffix -- `tail`.